scripts/test

```diff
@@ -5,8 +5,16 @@ cd $(dirname $0)/..
 
 echo Looking for packages to test
 
-PACKAGES=". $(find -name '*.go' | xargs -I{} dirname {} |  cut -f2 -d/ | sort -u | grep -Ev '(^\.$|.git|.trash-cache|vendor|bin|test)' | sed -e 's!^!./!' -e 's!$!/...!')"
+PACKAGES=". $(find . -path ./vendor -prune -o \
+                     -path ./test/e2e -prune -o \
```

Comment:
If/when we have Go files under `test` outside `test/e2e`, this will break, because the matching files under `test` will result in `test/...` being added to `PACKAGES`, which includes `test/e2e` again. I think we need to exclude `test` entirely for now.

Reply:
See #74 for an alternate proposal (untested on macOS).

Reply:
oh, right, you're correct.